.github/workflows/license.yaml

```diff
@@ -13,4 +13,4 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Check License Header
-      uses: apache/skywalking-eyes/header@df70871af1a8109c9a5b1dc824faaf65246c5236
+      uses: korandoru/hawkeye@v3
```

Comment:
What's the difference between these two apps? Like `skywalking-eyes` cannot add an empty line between license header and code?

Reply:
Mainly for https://github.com/apache/skywalking/issues/10223 and I like my solution on hawkeye.

Other reasons are not quite strong but if you don't have Golang env, when you use the docker image to apply or check, hawkeye's image takes ~30MB while skywalking-eyes take > 500MB.